src/em_keyboard/__init__.py

```diff
@@ -33,7 +34,9 @@
 
 __version__: str = importlib.metadata.version("em_keyboard")
 
-EMOJI_PATH = os.path.join(os.path.dirname(__file__), "emojis.json")
+with importlib.resources.path("em_keyboard", "emojis.json") as em_json:
```

Comment:
Python 3.11+ gives a DeprecationWarning for `path`. I could switch on the python version to use the two ways, but I don't know if that's necessary?

```.tox\py311\Lib\site-packages\em_keyboard\__init__.py:37
  .\em-keyboard\.tox\py311\Lib\site-packages\em_keyboard\__init__.py:37: 
DeprecationWarning: path is deprecated. Use files() instead. 
Refer to https://importlib-resources.readthedocs.io/en/latest/using.html#migrating-from-legacy for migration advice.
    with importlib.resources.path("em_keyboard", "emojis.json") as em_json:
```

Reply:
Yes please, let's remove the deprecation warning. We could also do `try: ... files() ... `/`except`.